ods_ci/tests/Tests/400__ods_dashboard/412__ods_dashboard_resources.robot

```diff
@@ -236,28 +228,27 @@ Verify Quick Starts Work As Expected When Restarted And Left In Between
         Run Keyword And Continue On Failure     Link Text On QuickStart Card Should Be  element=${element}
         ...    exp_link_text=Restart
         Open QuickStart Element In Resource Section By Name     ${element}
-        Page Should Not Contain         //article[@id="${element}"]//span[@class="pf-v5-c-label pf-m-green pf-m-outline"]
+        Page Should Not Contain    //article[@id="${element}"]//span[@class="pf-v5-c-label pf-m-green pf-m-outline"]
         ${count}=   Get The Count Of QuickStart Steps
         Run Keyword And Continue On Failure     Click Button    //button[@data-testid="qs-drawer-start"]
         Run Keyword And Continue On Failure     Wait Until Page Contains Element
         ...    //div[@class="pfext-quick-start-content"]
-        ${temp_count}   Set Variable    2
-        FOR     ${index}    IN RANGE    ${temp_count}
-            IF  ${index} != ${count-1}
-                Run Keyword And Continue On Failure     Wait Until Keyword Succeeds    2 times   0.3s
-                ...    Mark Step Check As Yes
-                Run Keyword And Continue On Failure     Go To Next QuickStart Step
-            END
+        ${half_count}=   Set Variable    ${count // 2}
```

Comment:
Would be nice if you could provide a brief description to this test (`[Description]`) to explain why we're doing this in two steps for cycles (0 - half ; half - count) for dummies like me :slightly_smiling_face:

Reply:
Added description, thanks!